launcher/modManager/cmodlistview_moc.cpp

```diff
@@ -243,8 +248,11 @@ QString CModListView::genModInfoText(CModEntry & mod)
 	result += replaceIfNotEmpty(mod.getValue("installedVersion"), lineTemplate.arg(tr("Installed version")));
 	result += replaceIfNotEmpty(mod.getValue("latestVersion"), lineTemplate.arg(tr("Latest version")));
 
-	if(mod.getValue("size").isValid())
-		result += replaceIfNotEmpty(CModEntry::sizeToString(mod.getValue("size").toDouble()), lineTemplate.arg(tr("Download size")));
+	if(mod.getValue("localSize").isValid())
+		result += replaceIfNotEmpty(CModEntry::sizeToString(mod.getValue("localSize").toDouble()), lineTemplate.arg(tr("Size")));
+	if((mod.isAvailable() || mod.isUpdateable()) && mod.getValue("size").isValid())
+		result += replaceIfNotEmpty(CModEntry::sizeToString(mbToBytes(mod.getValue("size").toDouble())), lineTemplate.arg(tr("Download size")));
```

Comment:
Perhaps rename "size" to "downloadSize" for clarity?

Also - validation schema needs update.

Reply:
Since it is existing but unused field, don't want to change it. But let me thing about it

> validation schema needs update.

will do

Reply:
Please consider rename.
"size" is very ambiguous. And unlike local size it needs to be set by modders.

`"size" : 2.5` is very unclear. Is this size of mod? Size of download? Something else? Should it be defined in megabytes? Kilobytes? Bytes?

Reply:
having downloadSizeMB is even less clear. People may think that if there is downloadSizeMB than could be possible downloadSizeKB. DownloadSize is better, but still size is pretty clear and simple and it present since 0.99 (I removed column when moved mods to GitHub), no need to overcomplicate

Reply:
> People may think that if there is downloadSizeMB than could be possible downloadSizeKB

This will be immediately detected by our validation. This is precisely why we have it.

> but still size is pretty clear and simple 

No it is no. At the very least it can also mean size of unpacked mod.